README.adoc

```diff
@@ -334,6 +333,10 @@ The `CYPRESS_RECORD_KEY` environment variable needs to be set for this to work.
 |===
 |Testcontainers-cypress |Testcontainers | Cypress
 
+|https://github.com/wimdeblauwe/testcontainers-cypress/releases/tag/testcontainers-cypress-1.9.0[1.9.0]
+|https://github.com/testcontainers/testcontainers-java/releases/tag/1.19.0[1.19.0]
```

Comment:
I see 1.19.1 is released yesterday. Maybe update here and in the `pom.xml` to that version?

Reply:
Done, thanks